app/models/case_information.rb

```diff
@@ -23,22 +25,67 @@ def latest_early_allocation
   validates :manual_entry, inclusion: { in: [true, false], allow_nil: false }
   validates :nomis_offender_id, presence: true, uniqueness: true
 
-  validates :local_divisional_unit, :team, presence: true, unless: ->{ manual_entry }
+  validates :team,
```

Comment:
I'm unsure about this validation. I think it needs a comment to explain why team is mandatory when we're not entering manually (i.e. from nDelius) and the probation service is (not?) Scot/NI - I'm finding it hard to understand the 'unless' inverted logic in this case. (Actually I think the logic is slightly wrong - I think the intention is to validate the team when manually entering English Welsh cases)

Reply:
It looks like the other side-effect of this change is that we lose the validation on `:local_divisional_unit` needing to be present.

Currently, the nightly Delius import fails if the offender is assigned to a team which doesn't have an LDU – this is covered by error code `DeliusImportError::MISSING_LDU`.

If I'm following this correctly, dropping the validation here will mean that is _will_ be possible to assign offenders (either manually or automatically) to a team which doesn't have an LDU.

Is that something we want to do?

As an aside: we may want to adjust the 'Teams' autocomplete list to filter out teams which don't have an LDU – so a manual entry can never trigger the 'missing LDU' error type.